pytorch_lightning/loggers/neptune.py

```diff
@@ -273,44 +273,53 @@ def __init__(
         super().__init__()
         self._log_model_checkpoints = log_model_checkpoints
         self._prefix = prefix
+        self._run_name = name
```

Comment:
Can't push to the branch for some reason. Can you add this code at line 269 please?
```py
        if neptune is None:
            raise ModuleNotFoundError(
                "You want to use the `Neptune` logger which is not installed yet, install it with"
                " `pip install neptune-client`."
            )
```

This should fix the remaining issue and I checked locally the tests are running. Thanks

Reply:
Thank you so much :wink:. I've updated the logger initializer as you wish and I confirm that it passes locally :wink: